src/app/controllers/RestaurantMenuController.js

```diff
@@ -0,0 +1,49 @@
+import { celebrate, Segments, Joi } from 'celebrate';
+import RestaurantMenu from '../schemas/RestaurantMenu';
+
+class RestaurantMenuController {
+	async createRestaurantMenu(req, res) {
```

Comment:
This function only keeps the restaurant menu of one week, right? Wouldn't it be cool if we stored the menu of all the weeks? This way we could also create a DailyMenuReview schema, or something like that, to store reviews for each of the RU's meals, linking each review to one entry of the RestaurantMenu schema.

Reply:
No, the idea is that the menus will be saved forever. I updated the PR description with more information. About garnish and vegetarianOption, I followed this image from the official app, it looks like they don't change from lunch to dinner, but if they do, I think we can add a new field to separate, transform it into an array will be confusing. 

![screen-4](https://user-images.githubusercontent.com/24527951/79672427-2dbbe980-81a8-11ea-8d4a-3ee5ab2ea35b.jpg)

Reply:
Oh, ok. I thought the menu was saved for only one week because of the 'day' on line 7. It sounded as 'day of the week', for me, not as a unique date. About garnish and vegetarianOption, ok! Let's wait and see how the menu is going to look like after the pandemic.